plugins/chat/db/migrate/20240827040550_copy_chat_mention_notifications_notification_id_values.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class CopyChatMentionNotificationsNotificationIdValues < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    execute("DROP INDEX IF EXISTS chat_mention_notifications_tmp_migration_index")
+
+    execute(<<~SQL)
+      CREATE INDEX #{Rails.env.test? ? "" : "CONCURRENTLY"} chat_mention_notifications_tmp_migration_index ON chat_mention_notifications (notification_id)
+      WHERE notification_id != new_notification_id
+    SQL
+
+    sql = <<~SQL
+      UPDATE chat_mention_notifications
+      SET new_notification_id = notification_id
+      WHERE notification_id IN (
+        SELECT
+          notification_id
+        FROM chat_mention_notifications
+        WHERE notification_id != new_notification_id
+        LIMIT 100000
+      )
+    SQL
```

Comment:
There is no `id` column on the `chat_mention_notifications` table so we have to loop here.

Reply:
Can we loop over `notification_id`?

Reply:
Given that `notification_id` is unique, I think it might be possible. I'll update the PR tomorrow.